src/hooks/useInitializeCollectionDraft.ts

```diff
@@ -57,12 +57,15 @@ function useInitializeCollectionDraft() {
 
     const resetBindingsEditorState = useBindingsEditorStore_resetState();
 
-    // Draft Editor Store
+    // Global Draft Editor Store
     const draftId = useEditorStore_persistedDraftId();
     const setDraftId = useEditorStore_setId();
 
     const setPersistedDraftId = useEditorStore_setPersistedDraftId();
 
+    // Local Draft Editor Store
+    const setLocalDraftId = useEditorStore_setId({ localScope: true });
+
```

Comment:
Nothing to change... but this _feels_ weird to me... kind of like a code smell or something. Where we are fetching stuff for the global version of a store and then also fetching from the local copy. Cannot think of a reason why this is bad just feels wonky to me.